web/src/components/Plugin/service.ts

```diff
@@ -34,6 +34,7 @@ export const fetchList = () => {
     const typedData = data.data.map(item => ({
       ...item,
       type: PLUGIN_LIST[item.name]?.type || "other",
```

Comment:
Does `type` props still be used?

Reply:
need @LiteSun to have a search

Reply:
> Does type props still be used? If not we can delete it ^^

Type props still are used.